sdk/tests/deqp/framework/opengl/gluShaderUtil.js

```diff
@@ -37,6 +37,15 @@ gluShaderUtil.GLSLVersion = {
 };
 
 /**
+ * gluShaderUtil.glslVersionUsesInOutQualifiers
+ * @param {gluShaderUtil.GLSLVersion} version
+ * @return {boolean}
+ */
+gluShaderUtil.glslVersionUsesInOutQualifiers = function(version) {
+    return version == gluShaderUtil.GLSLVersion.V300_ES;
```

Comment:
It might be more future-proof to say "version != gluShaderUtil.GLSLVersion.V100_ES".